src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -100,6 +100,16 @@ protected ActionResult execute() {
             data.emailToEdit.emailId = emailId;
         }
 
+        if (addressModeOn && groupModeOn) {
+            statusToAdmin = "Email queued for sending." + "<br/>" + "Receipient: " + addressReceiverListString
+                            + "<br/>" + "Group receiver's list " + groupReceiverListFileKey;
+            statusToUser.clear();
+
+            statusToUser.add(new StatusMessage("Email will be sent within an hour to "
```

Comment:
Actually, I am ok with showing two status messages (one for groupMode, one for addressMode) @Neurrone What do you think?